pkgs/development/python-modules/django-admin-sortable2/default.nix

```
@@ -8,15 +8,15 @@
 
 buildPythonPackage rec {
   pname = "django-admin-sortable2";
-  version = "2.2.1";
+  version = "2.2.2";
   format = "setuptools";
```

```suggestion
  pyproject = true;
```
and add `build-system = [ setuptools ]`